include/mbedtls/ssl_internal.h

```diff
@@ -658,6 +661,11 @@ struct mbedtls_ssl_handshake_params
 
     mbedtls_ssl_tls_prf_cb *tls_prf;
 
+    /* Buffer holding the digest up to, and including, 
+     * the Finished message sent by the server. 
+     */
+    unsigned char digest[MBEDTLS_MD_MAX_SIZE];
```

Comment:
What about a more accurate name like `server_finished_digest` instead?

Reply:
I have changed the name of the variable to server_finished_digest.